exec/build-modules/src/index.ts

```diff
@@ -53,6 +54,7 @@ export async function buildModules (
     warn,
   }
   const chunks = buildSequence(depGraph, rootDepPaths)
+  const allowBuild = opts.allowBuild ?? (() => true)
```

Comment:
Nice. I like the abstraction of `allowBuild` to avoid propagating the concept of `onlyBuiltDependenciesFile` down to this lower level component.